packages/crayons-icon/icon-utils/icon-registry.js

```diff
@@ -0,0 +1,10 @@
+export const registerIconLibrary = (items) => {
+  var elements = document.querySelectorAll('fw-icon');
```

Comment:
Why are we doing a querySelector to infer if the lib's used?
Ideally, this should be done at the component level right?